Web/amazon-connect-ccp-web-calling-example/src/components/MyCustomCCP/ConferenceView/DisplayPanel/ControlBar/VideoButton/VideoButton.js

```diff
@@ -26,7 +26,7 @@ class VideoButton extends HTMLElement {
   }
 
   updateLabel() {
-    this.label = this.label === 'Turn on video' ? 'Turn off video' : 'Turn on video';
+    this.label = this.label === 'Start video' ? 'Stop video' : 'Start video';
```

Comment:
Is there a way to update the label based on the video state instead?

Reply:
Yes, for a fully functional application, we should check if button's onClick function succeeds or not and then have a flag to track this state and update the UI accordingly. But we decided not to do such error handling in this sample App to keep it simple. The main function toggleVideo and toggleScreenSharing is very unlikely to fail. I've tried a lot and didn't run into that. The main idea of this sample is to show case Netra related APIs in stream js and then chime sdk.